adapter/config/tpr/client.go

```diff
@@ -187,7 +188,7 @@ func (cl *Client) RegisterResources() error {
 		for _, kind := range kinds {
 			list := &ConfigList{}
 			err := cl.dynamic.Get().
-				Namespace(api.NamespaceAll).
+				Namespace(v1.NamespaceAll).
```

Comment:
Can we create an alias for this modules? We cannot change the code every time there is a new version.